src/main/resources/templates/db_scripts/budget_encumbrances_rollover.sql

```diff
@@ -138,9 +139,10 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.rollover_order(_order_id
                         'amountAwaitingPayment', 0,
                         'amountExpended', 0,
                         'amountCredited', 0,
-                        'status', CASE WHEN (tr.jsonb->'encumbrance'->>'reEncumber')::boolean
-                                      THEN 'Unreleased'
-                                      ELSE 'Released'
+                        'status', CASE WHEN input_encumbrancesRolloverLen > 0 THEN
+                                    CASE WHEN (tr.jsonb->'encumbrance'->>'reEncumber')::boolean THEN 'Unreleased' ELSE 'Released' END
+                                  ELSE
+                                    'Released'
                                   END
```

Comment:
This SQL is already present in `schema.json` for update during the module installation: 
![image](https://github.com/user-attachments/assets/406b1401-7317-4a0a-9b37-16ef7a5deb73)